spec/support/features/course_details_steps.rb

```diff
@@ -33,6 +34,14 @@ def given_a_course_is_available_for_selection
       create(:course_with_subjects, accredited_body_code: trainee.provider.code, route: trainee.training_route)
     end
 
+    def given_subject_specialisms_are_available_for_selection
+      create(:subject_specialism, name: subject_specialism_name)
+    end
+
+    def subject_specialism_name
+      Dttp::CodeSets::AllocationSubjects::MAPPING.values.first[:subject_specialisms].first.capitalize
```

Comment:
You could just use the name directly `Dttp::CodeSets::CourseSubjects::{constant_name}`